src/rules/no_empty_function.rs

```diff
@@ -0,0 +1,26 @@
+// Copyright 2020 the Deno authors. All rights reserved. MIT license.
+use super::Context;
+use crate::traverse::AstTraverser;
+use swc_ecma_ast::FnDecl;
+
+pub struct NoEmptyFunction {
+  context: Context,
+}
+
+impl NoEmptyFunction {
+  pub fn new(context: Context) -> Self {
+    Self { context }
+  }
+}
+
+impl AstTraverser for NoEmptyFunction {
+  fn walk_fn_decl(&self, fn_decl: FnDecl) {
+    if fn_decl.function.body.unwrap().stmts.is_empty() {
```

Comment:
It will crash if `function.body` is None

Reply:
so should I ban both "empty body" and "no body"?